tests/generator-Tests/Unit-Tests/StringOverloads.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using NUnit.Framework;
+
+namespace generatortests
+{
+	[TestFixture]
+	public class StringOverloads : BaseGeneratorTest
```

Comment:
This style of integration test is old, slow, and requires a lot of extra ceremony.  We prefer to use pure unit tests instead of these, you should be able to write a single method test in the style of:

https://github.com/xamarin/java.interop/blob/main/tests/generator-Tests/Unit-Tests/CodeGeneratorTests.cs#L178